File: react-native-hoppr-ads/ios/ReactNativeHopprAds/HierarchyReaderModule.m
//
//  HierarchyReaderModule.m
//  DemoReactNativeSdk
//
//  Created by Maxence Bourdin on 26/4/2023.
//

#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>
#import <React/RCTEventEmitter.h>

@interface RCT_EXTERN_MODULE(HierarchyReaderModule, NSObject)
  RCT_EXTERN_METHOD(getViewHierarchy: (RCTResponseSenderBlock *)successCallback)
@end

File: react-native-hoppr-ads/ios/ReactNativeHopprAds/HierarchyReaderModule.swift
//
//  HierarchyReaderModule.swift
//  DemoReactNativeSdk
//
//  Created by Maxence Bourdin on 26/4/2023.
//

import Foundation
import UIKit
import AVFoundation

@objc(HierarchyReaderModule)
class HierarchyReaderModule: NSObject {
  
  @objc
  func getViewHierarchy(_ successCallback: RCTResponseSenderBlock) {
    successCallback(["hello"])
  }
  
  // override static func requiresMainQueueSetup() -> Bool {
  //   return true
  // }
}

